capstone/src/main/java/com/google/sps/data/ReadData.java

```diff
@@ -25,13 +27,24 @@ public void readCSV() {
 
       while ((line = reader.readLine()) != null) {
         String[] values = line.split(DELIMITER);
-        String id = values[0];
-        double score = Double.parseDouble(values[2]);
-        String gender = values[3];
-        String ageRange = values[4];
-        long responseTime = Long.parseLong(values[5]);
-        String date = values[6];
-        long day = Long.parseLong(values[7]);
+        String id = values[0] + "-" + values[1];
+        SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+        Date date;
+
+        try {
+          date = formatter.parse(values[2]);
+        } catch (ParseException e) {
+          date = null;
+        }
+        String completion = values[3];
+        String gender = values[5];
+        String ageRange = values[6];
+        String answerOne = values[8];
+        String answerTwo = values[9];
+        double score = Double.parseDouble(values[10]);
+        long responseTimeOne = Long.parseLong(values[11]);
```

Comment:
What are the difference between these response times? Can you add some context here? (similar thoughts to the other variables named like this .*One, .*Two,...).

Reply:
I'll add documentation